src/py/pyodide/_package_loader.py

```diff
@@ -54,9 +54,9 @@ def unpack_buffer(
     buffer: JsProxy,
     *,
     filename: str = "",
-    format: str = None,
-    target: Literal["site", "lib", None] = None,
-    extract_dir: str = None,
+    format: str | None = None,
```

Comment:
Actually the implicit optional is maybe not so bad.

Reply:
It's better to be explicit (plus it's required if we want to eventually go to strict mode).

Reply:
In this case, for example, format can be None - either now or later. The type is `str | None` in either case - mypy is just smart enough to inject the `| None` for you. But other tools might not, a reader might not notice the `= None` part when reading the type, simply changing the default value changes the type, the type is incorrect if you try to copy it to a `.pyi` file (since the `= None` becomes `= ...`), etc. That's why implicit optionals were [removed from PEP 484](https://peps.python.org/pep-0484/#union-types). (There's a long discussion on it somewhere which I didn't bother to find the link for).

Reply:
OK, you convinced me that this is better :)